content/html/concepts/headings/headings.md

```diff
@@ -22,8 +22,15 @@ Among the HTML elements are the 6 heading elements. They are labeled 1-6, with 1
 <h5>Heading 5</h5>
 <h6>Heading 6</h6>
 ```
-
-![Rendered page with the 6 HTML heading elements](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-headings.png)
+<details>
+<summary>Headings</summary>
+# Heading 1
+## Heading 2
+### Heading 3
+#### Heading 4
+##### Heading 5
+###### Heading 6
```

Comment:
I don't think this is rendering what the image was displaying? This worked for me, though: 

```html
<details>
<summary>Headings</summary>
<h1>Heading 1</h1>
<h2>Heading 2</h2>
<h3>Heading 3</h3>
<h4>Heading 4</h4>
<h5>Heading 5</h5>
<h6>Heading 6</h6>
</details>
```

Reply:
@Dusch4593 Got it. I made the change